solidity/test/Converter.js

```diff
@@ -91,6 +112,24 @@ contract('Converter', accounts => {
         return converter;
     };
 
+    const createAnchor = async (type) => {
+        switch (type) {
+            case 0: 
+                anchor = await SmartToken.new('Token1', 'TKN1', 2);
+                break;
+
+            case 1:
+                anchor = await SmartToken.new('Pool1', 'POOL1', 2);
+                break;
+
+            case 2:
+                anchor = await PoolTokensContainer.new('Pool', 'POOL', 2);
```

Comment:
Forgot `'2'` at the end of the name and at the end of the symbol.

Reply:
I think...

Reply:
no `2` here - the pool tokens are gonna have 1 & 2.